### PR TITLE
Abo leeh

### PR DESCRIPTION
**Pull request by Yahya-Ehab**
added the basic GUI implementation

### CODE REVIEW

**Review by youssefkhalil320 — COMMENTED** [review]
LGTM!

File: src/GUI.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Apply a theme
+style = ttk.Style()
+style.theme_use('clam')  # or 'alt', 'default', 'classic', 'vista', 'xpnative'
+
+# Customize button and label styles
+style.configure('TButton', font=('Arial', 10), background='#E1E1E1')
+style.configure('TLabel', font=('Arial', 10), background='#F5F5F5', foreground='#333333')
+
+root = tk.Tk()
+root.title("Abo-Leeh Password Generator")
```
Comment (youssefkhalil320) [inline src/GUI.py:22]: What is Abo-Leeh ;)

File: src/GUI.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+root.configure(bg='#F5F5F5')  # Set background color
+
+# Set a fixed window size
+root.geometry('400x200')
```
Comment (youssefkhalil320) [inline src/GUI.py]: I guess it is better to make 40x200 a global variable

**Review by youssefkhalil320 — CHANGES_REQUESTED** [review]

File: src/GUI.py (hunk 1 above)
Comment (Yahya-Ehab) [reply src/GUI.py:22]: the nickname my dad used to give for my grandpa lol

Comment (youssefkhalil320) [reply src/GUI.py:22]: nice!!

Comment (Yahya-Ehab) [reply src/GUI.py:22]: thanks :)

File: src/GUI.py (hunk 2 above)
Comment (Yahya-Ehab) [reply src/GUI.py]: done

**Pull request merged**